modules/new_serverpod_auth/serverpod_auth_session/serverpod_auth_session_flutter/lib/src/key_value_storage.dart

```diff
@@ -0,0 +1,10 @@
+import 'dart:async';
+
+/// String-based key/value store.
+abstract class KeyValueStorage {
+  /// Gets the stored value for [key].
+  FutureOr<String?> get(String key);
+
+  /// Sets [key] to the new [value].
+  FutureOr set(String key, String? value);
```

Comment:
```suggestion
  FutureOr<void> set(String key, String? value);
```
.. as opposed to `dynamic`

Reply:
Interesting that the default client (Flutter) linter config from us does not include the "no implicit dynamic" one.

I was so used to that always being active I forgot that forgetting a "open" type annotation was even a thing anymore.

Will fix.